config/configgrpc/configgrpc.go

```diff
@@ -226,6 +226,10 @@ func (gcs *GRPCClientSettings) ToDialOptions(ext map[config.ComponentID]componen
 		opts = append(opts, grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingPolicy":"%s"}`, gcs.BalancerName)))
 	}
 
+	// Enable OpenTelemetry observability plugin.
+	opts = append(opts, grpc.WithUnaryInterceptor(otelgrpc.UnaryClientInterceptor()))
+	opts = append(opts, grpc.WithStreamInterceptor(otelgrpc.StreamClientInterceptor()))
+
```

Comment:
This enables context propagation, right? So, we will propagate context for grpc-based protocols, like OTLP. Is this something we want?

Reply:
This is similar with what we had before with opencensus using the stats handler.

Reply:
Is it not the case that w3c is the default?

Reply:
No, unfortunately it isn't.

> The OpenTelemetry API MUST use no-op propagators unless explicitly configured otherwise.
- [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#global-propagators)